sonar-plugin-api/src/main/java/org/sonar/api/issue/Issue.java

```diff
@@ -176,6 +176,10 @@
   @CheckForNull
   String authorLogin();
 
+  /**
+   * @deprecated since 5.5 Action plans are dropped in 5.5. This field has no effect
+   */
+  @Deprecated
   @CheckForNull
   String actionPlanKey();
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)